packages/eas-cli/src/credentials/manager/SelectAndroidBuildCredentials.ts

```diff
@@ -0,0 +1,67 @@
+import { AndroidAppBuildCredentialsFragment } from '../../graphql/generated';
+import { promptAsync } from '../../prompts';
+import { promptForNameAsync } from '../android/actions/BuildCredentialsUtils';
+import { AppLookupParams } from '../android/api/GraphqlClient';
+import { AndroidAppBuildCredentialsMetadataInput } from '../android/api/graphql/mutations/AndroidAppBuildCredentialsMutation';
+import { sortBuildCredentials } from '../android/utils/printCredentialsBeta';
```

Comment:
Maybe move this function somewhere else. `printCredentialsBeta` doesn't seem to be a place where I'd be looking for a function that sorts credentials :D